ccd-definition/CaseEventToFields/RequestExtension.json

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "REQUEST_EXTENSION",
+    "CaseFieldID": "extensionProposedDeadline",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "ProposeDeadline",
+    "PageLabel": " ",
```

Comment:
let's just remove the pageLabel line if it is not defined as anything, makes the json a little cleaner

Reply:
same for other places in this file too